src/ripple/app/misc/AmendmentTable.h

```diff
@@ -154,10 +211,40 @@ class AmendmentTable
     doVoting (Ledger::ref lastClosedLedger, SHAMap::ref initialPosition) = 0;
 };
 
-std::unique_ptr<AmendmentTable>
-make_AmendmentTable (std::chrono::seconds majorityTime, int majorityFraction,
-    beast::Journal journal);
+/**
+   AmendmentTableInjections is used to insert moc objects into the amendment
```

Comment:
Hi Scott, no objection here.  But just for my own curiosity, I've always seen it spelled "mock", like mock turtle.  Is "moc" also a common spelling?